events/config.go

```diff
@@ -0,0 +1,74 @@
+package events
+
+import (
+	"errors"
+	"github.com/Shopify/sarama"
+	"github.com/kelseyhightower/envconfig"
+)
+
+const DeadLetterSuffix = "-dead-letters"
+
+type CloudEventsConfig struct {
+	EventSource           string   `envconfig:"CLOUD_EVENTS_SOURCE" required:"true"`
+	KafkaBrokers          []string `envconfig:"KAFKA_BROKERS" required:"true"`
+	KafkaConsumerGroup    string   `envconfig:"KAFKA_CONSUMER_GROUP" required:"false"`
```

Comment:
Should there be a default consumer group?

Reply:
This might cause issues if two services are reading from the same topic. We could default to `EventSource` (which most of the times should be set to the service name), but I prefer explicitly setting the consumer group.